audio_synthesizer.py

```diff
@@ -114,7 +114,7 @@ def synthesize_mixtures(self):
                             ntraj = int(mixture_nm['trajectory'])
                             
                         # load event audio and resample to match RIR sampling
-                        eventsig, fs_db = soundfile.read(self._db_path + '/' + filename.split('/')[-1])
+                        eventsig, fs_db = soundfile.read(self._db_path + '/' + filename)#.split('/')[-1])

```

Comment:
I knew we were gonna run into troubles because of this line. The root of our disagreement is due to the fact that I'm using FSD50K and you are using NIGENS. That's cool though, as this code base should work with either. 

For FSD50K, the `db_config_fsd.obj` that this generator comes with an FSD50K path structure that needs to be generated using the annotations after downloading FSD50K. I got lazy and skipped this step. Maybe I should fix it. 
As a workaround, could you create a flag to check for NIGENS vs FSD50K for now?

Reply:
Added a switch case for `nigens` and `fsd50k`, the parameters inherited from `generation_parameters.py` contain info on the sound event database, so this is pretty straightforward. Should be safe to push now afaik.